### PR TITLE
ADD change version pointed by the RTD badge during version stepping process

### PR DESCRIPTION
**Pull request by fgalan**
Implements #1858 

@jmcanterafonseca (as expert in RTD and badgets)
@kzangeli (as father of the release.sh script)

### CODE REVIEW

File: scripts/build/release.sh [hunk 1]
```diff
@@ -153,6 +153,16 @@ mv /tmp/version.h              src/app/contextBroker/version.h
 rm -rf CHANGES_NEXT_RELEASE
 touch CHANGES_NEXT_RELEASE
 
+# Adjust Readthedocs documentation badge. Note that the procedure is not symetric (like in version.h), as
```
Comment (kzangeli) [inline scripts/build/release.sh]: This is just a comment, but anyway ...

Readthedocs -> read-the-docs ?   (looks better IMO)
symetric -> symmetric   (or perhaps symmetrical)

**Comment by jmcanterafonseca** [comment]
LGTM, thanks

File: scripts/build/release.sh (hunk 1 above)
Comment (fgalan) [inline scripts/build/release.sh]: Typo fixed in last commit (sorry, at mobile it is difficult to paste hash).

Regarding readthedocs, if @jmcanterafonseca hasn't raised comment I think is ok.

**Comment by kzangeli** [comment]
LGTM

**Pull request merged**